plugins/graphql-voyager/README.md

```diff
@@ -0,0 +1,75 @@
+# graphql-voyager
+
+Welcome to the graphql-voyager plugin!
+
+_This plugin was created through the Backstage CLI_
+
+## Getting started
+
+### Installing
+
+To get started, first install the plugin with the following command:
+
+```bash
+# From your Backstage root directory
+yarn add --cwd packages/app @backstage/plugin-graphql-voyager
+```
+
+### Adding the page
+
+In order to be able to navigate to the graphQL voyager page, a new route needs to be added to the app. This can be done in the following way:
+
+```tsx
+// packages/app/App.tsx
+
+  import { GraphQLVoyagerPage } from "@backstage/plugin-graphql-voyager";
+
+  const routes = (
+    <FlatRoutes>
+      <Route path="/graphql-voyager" element={<GraphqlVoyagerPage/>}/>
+```
+
+### Configuration
+
+In order for the plugin to function correctly. GraphQL endpoints need to be added / configured through the GraphQLVoyager API. This can be done by implementing the `graphQlVoyagerApiRef` exported by this plugin.
+
+Add your own configuration to the `packages/app/src/apis.ts` file the following way:
+
+```ts
+import { identityApiRef } from './IdentityApi';
+import { GraphQLVoyagerEndpoints } from './GraphQLVoyagerEndpoints';
+
+export const apis: AnyApiFactory[] = [
+  createApiFactory({
+    api: graphQlVoyagerApiRef,
+    deps: { identityApi: identityApiRef },
+    factory: ({ identityApiRef }) => {
+      GraphQLVoyagerEndpoints.from([
+        {
+          id: `graphql-voyager-endpoint-id`,
+          title: 'endpoint-title',
+          introspectionErrorMessage:
```

Comment:
hmm yes that could be, however the introspection method does not always error out. The reason why I have it in there is because whenever an endpoint can't be reached (wrong JWT token for example) the introspection method does not see this as an error, but just returns OK. 

Which causes the <Voyager/> component to crash, since it expects a .`data` property in the introspection result. 

Thats why I have added that custom errorMessage incase that happens

Reply:
I'd have thought that the introspection surely should have found that to be an error, sure sounds critical! Well ok.

Reply:
Totally agree with you on that! Seems kind of weird to not error out on that. I'll see if I can open a issue on their API for this :-)